src/documentation/how_tos/studio/how_to_execute_a_benchmark.ipynb

```diff
@@ -0,0 +1,113 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uuid import uuid4\n",
+    "\n",
+    "from documentation.how_tos.example_data import (\n",
+    "    EXAMPLE_1_INPUT,\n",
+    "    DummyAggregationLogic,\n",
+    "    DummyEvaluationLogic,\n",
+    "    DummyExample,\n",
+    "    DummyTask,\n",
+    ")\n",
+    "from intelligence_layer.connectors.studio.studio import StudioClient\n",
+    "from intelligence_layer.evaluation.benchmark.studio_benchmark import (\n",
+    "    StudioBenchmarkRepository,\n",
+    ")\n",
+    "from intelligence_layer.evaluation.dataset.studio_dataset_repository import (\n",
+    "    StudioDatasetRepository,\n",
+    ")\n",
+    "\n",
+    "examples = [\n",
+    "    DummyExample(input=\"input0\", expected_output=\"expected_output0\", data=\"data0\"),\n",
+    "    DummyExample(\n",
+    "        input=EXAMPLE_1_INPUT, expected_output=\"expected_output1\", data=\"data1\"\n",
+    "    ),\n",
+    "]"
```

Comment:
We need  `load_dotenv()` after the imports

Reply:
did this